sections/semantics-grouping-content.include

```diff
@@ -42,8 +42,8 @@
   The <{p}> element should not be used when a more specific element, such as a list, is more appropriate.
 
   <div class="note">
-    Many elements whose typical <a>rendering</a> is a block cannot be children of <{p}> elements,
-    because when the parser finds them it automatically closes the <{p}> element - effectively inserting a <code>&lt;/p&gt;</code> tag.
+    Many elements whose typical rendering is a block, cannot be children of <{p}> elements.
```

Comment:
I would remove the comma too, but that's pretty editorial...

Reply:
Done.